lib/swig/structs.i

```diff
@@ -0,0 +1,252 @@
+typedef struct{
```

Comment:
Do not duplicate these . Use [SWIG `%include` directive](http://www.swig.org/Doc1.3/Preprocessor.html#Preprocessor_nn2) .

Put this in `skycoin.cipher.encrypt.i`